packages/sys-apps/cpuid/cpuid-20201006.exheres-0

```diff
@@ -17,12 +17,17 @@ MYOPTIONS="parts: binaries documentation"
 
 DEPENDENCIES=""
 
-BUGS_TO="virkony@gmail.com cpuid@etallen.com"
+DEFAULT_SRC_COMPILE_PARAMS=(
+    ${PN} cpuinfo2cpuid.man
```

Comment:
True, using explicit `make` targets is much nicer and allows to build everything needed without `sed`ing things (only these here: cpuifn2ocpuid is a perl script therefore does not require building, cpuid.man already comes prewritten)